fstest/fstests/bits.go

```diff
@@ -0,0 +1,33 @@
+//+build !go1.9
+
+package fstests
+
+func leadingZeros64(x uint64) int {
```

Comment:
How is this different from math.LeadingZeros64?

Reply:
The [`bits.LeadingZeros64`](https://golang.org/pkg/math/bits/#LeadingZeros64) was added in go1.9, so a backport was needed for the tests.

The official version is probably faster, but performance is not key for the usage.

Reply:
Ah, that makes sense!